lib/common/socket/evloop.c.h

```diff
@@ -624,14 +624,12 @@ int h2o_evloop_run(h2o_evloop_t *loop, int32_t max_wait)
         h2o_linklist_t expired;
         h2o_linklist_init_anchor(&expired);
         h2o_timerwheel_get_expired(loop->_timeouts, loop->_now_millisec, &expired);
-        if (h2o_linklist_is_empty(&expired))
-            break;
```

Comment:
I'm not sure if we can remove this if statement, as it is the only way of exiting from the `while (1)` loop that surrounds.

Reply:
Aha! I totally misread the code and didn't realize this `break` was to exit from the outer loop. I'll close this PR.